apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.html.ts

```diff
@@ -203,21 +213,67 @@ function FeatureGrantBreadcrumbs({ featureGrant }: { featureGrant: FeatureGrantR
 function FeatureGrant({
   featureGrant,
   overridden,
+  csrfToken,
 }: {
   featureGrant: FeatureGrantRow;
   overridden: boolean;
+  csrfToken: string;
 }) {
   return html`
     <div
-      class="list-group-item d-flex flex-row align-items-center ${overridden ? 'text-muted' : ''}"
+      class="list-group-item d-flex flex-column flex-md-row flex-nowrap align-items-md-center ${overridden
+        ? 'text-muted'
+        : ''}"
     >
       <div>${FeatureGrantBreadcrumbs({ featureGrant })}</div>
+      <div class="ml-auto d-flex flex-row flex-nowrap flex-shrink-0">
+        <form
```

Comment:
I probably went a bit overboard with HTMX niceness here, but I figured this was good practice and gives us some nice prior art for making more valuable pages nicer in the future.

Reply:
It certainly produces a very nice UX ❤️